### DISCUSSION TITLE
Data Safety issue

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#871] [Q&A]
Data Safety issue :-
No Login sign up used
No online database
Only admob advertising
What's the solution I tried many way in console it doesn't not working 
![IMG-20230515-WA0036](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/37414496/c6f25681-ac50-47e2-a91e-103875f7d37a)

**Comment by PranavPurwar** [comment]
what ways, we can't just guess

**Comment by subhash16** [comment]
How to solv this

**Comment by subhash16** [comment]
PRANAV help Me

**Comment by PranavPurwar** [comment]
what ways did you try to fix it?

**Comment by subhash16** [comment]
I fill data safety

**Comment by subhash16** [comment]
What is the solution

**Comment by subhash16** [comment]
![Screenshot_2023-05-15-18-11-54-622_com sketchware remod](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/37414496/5d6f0a21-d07f-4f44-8706-479af2c4df58)
Also try this

**Comment by subhash16** [comment]
What kinda problem is this not solving

**Comment by subhash16** [comment]
Problem is my side
